xapian-core/tests/harness/backendmanager.cc

```diff
@@ -302,6 +302,14 @@ BackendManager::clean_up()
 {
 }
 
+bool
+BackendManager::kill_server(const std::string& uuid)
+{
+    throw Xapian::InvalidOperationError("kill_server called for " +
+    uuid + ", but available only for remote servers");
+    return false;
```

Comment:
There's no point calling `return` after an unconditional `throw` as it can never be reached.